python/cuml/pipeline/__init__.py

```diff
@@ -19,14 +19,17 @@
 if has_sklearn():
     from sklearn.pipeline import Pipeline, make_pipeline
 
-disclaimer = """
-This code is developed and maintained by scikit-learn and imported
-by cuML to maintain the familiar sklearn namespace structure.
-cuML includes tests to ensure full compatibility of these wrappers
-with CUDA-based data and cuML estimators, but all of the underlying code
-is due to the scikit-learn developers.\n\n"""
+    disclaimer = """
+    This code is developed and maintained by scikit-learn and imported
+    by cuML to maintain the familiar sklearn namespace structure.
+    cuML includes tests to ensure full compatibility of these wrappers
+    with CUDA-based data and cuML estimators, but all of the underlying code
+    is due to the scikit-learn developers.\n\n"""
 
-Pipeline.__doc__ = disclaimer + Pipeline.__doc__
-make_pipeline.__doc__ = disclaimer + make_pipeline.__doc__
+    Pipeline.__doc__ = disclaimer + Pipeline.__doc__
+    make_pipeline.__doc__ = disclaimer + make_pipeline.__doc__
 
-__all__ = ['Pipeline', 'make_pipeline']
+    __all__ = ['Pipeline', 'make_pipeline']
+else:
+    raise ImportError("Scikit-learn is needed to use "
```

Comment:
If we're still raising an ImportError, this won't resolve #5011, right? We need something else to indicate that the Pipeline class is not available.

Reply:
Oh, this is actually fine! I thought we were importing this file into an `__init__.py` elsewhere, but that does not appear to be the case. LGTM!